Vienna/Sources/Main window/WebKitArticleTab.swift

```diff
@@ -55,6 +55,25 @@ class WebKitArticleTab: BrowserTab, ArticleContentView, CustomWKUIDelegate {
         self.webView.contextMenuProvider = self
     }
 
+    // handle special keys when the article view has the focus
+    override func keyDown(with theEvent: NSEvent) {
+        guard let theString = theEvent.characters else {
+            return
+        }
+        if theString.count == 1 {
+            let keyChar = (theString as NSString).character(at: 0)
+            if NSApp.appController.handleKeyDown(keyChar, withFlags: theEvent.modifierFlags.rawValue) {
+                return
+            }
+            // Don't go back or forward in article view.
+            if ((theEvent.modifierFlags.rawValue & NSEvent.ModifierFlags.command.rawValue) != 0)
+            && ((keyChar == NSLeftArrowFunctionKey) || (keyChar == NSRightArrowFunctionKey)) {
+                return
+            }
```

Comment:
I don´t understand how cmd+back/forward arrow would trigger back/forward on the webview, since this view is just a wrapper around a webview. If that really happens, I would like to understand the mechanism. If it doesn´t, that event should not be intercepted here.

Reply:
You're right. I just transposed the existing code in TabbedWebView.m, but it wasn't necessary.